apps/settings/js/panels/about_more_info/hardware_info.js

```diff
@@ -52,6 +52,8 @@ define(function(require) {
      */
     _refreshBluetoothAddress: function mi__refreshBluetoothAddress(address) {
       // update UI fields
+      if (address===null || address==="")
+        address="Bluetooth is disabled.";
       for (var i = 0, len = this._elements.fields.length; i < len; i++) {
```

Comment:
since we only have one address field here, you can change the query sentence in `about_more_info/panels` to turn `fields` to bluetoothAddress or something else.